projects/packages/videopress/src/client/admin/components/pagination/stories/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useArgs } from '@storybook/client-api';
+import Pagination from '..';
+import Doc from './Pagination.mdx';
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+
+export default {
+	title: 'Packages/VideoPress/Pagination',
+	component: Pagination,
+	parameters: {
+		docs: {
+			page: Doc,
+		},
+	},
+} as ComponentMeta< typeof Pagination >;
+
+const Template: ComponentStory< typeof Pagination > = args => {
+	const [ , updateArgs ] = useArgs();
+	const onChangePage = ( newPage: number ) => {
+		updateArgs( { currentPage: newPage, disabled: true } );
```

Comment:
I suggest triggering an `action` in the component story. It's doable with something like...

```suggestion
		updateArgs( { currentPage: newPage, disabled: true } );
		action( 'onPageChange' )( newPage );
```

Don't forget to import the function:

```es6
import { action } from '@storybook/addon-actions';
```

https://user-images.githubusercontent.com/77539/186965084-9970313f-406e-4e5b-9796-cfd37e46e64d.mov